frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -1210,6 +1242,12 @@ export const dashboardLogic = kea<dashboardLogicType>([
                 actions.setShouldReportOnAPILoad(true)
             }
         },
+        abortQuery: ({ queryId }) => {
+            const { currentTeamId } = values
+            if (values.featureFlags[FEATURE_FLAGS.CANCEL_RUNNING_QUERIES]) {
+                api.create(`api/projects/${currentTeamId}/insights/cancel`, { client_query_id: queryId })
```

Comment:
Missing await?

Reply:
We don't await in the insightLogic either. I think it's OK here to be completely fire and forget.

Reply:
I think it is good hygene to always await. PTSD from untracked promises causing trouble in plugin-server land.